### ISSUE TITLE
When set WEBSOCKET_CORE_SUBSCRIBE='true', cannot start directus

### ISSUE THREAD

**Issue by loongmxbt** [br41nslug/directus-websocket-subscribe#21]
Hi, when I add `WEBSOCKET_CORE_SUBSCRIBE='true'` in my env, 
```
[16:43:42.717] INFO: Websocket listening on ws://localhost:8082/websocket
/Users/dragonszy/Github/elecsim/elecsim-pg-dapi/node_modules/.store/directus-extension-websocket-subscribe@3.0.10/node_modules/directus-extension-websocket-subscribe/dist/index.js:1

TypeError: Cannot read properties of undefined (reading 'buildDispatcher')
```

The following .env can start directus
```
WEBSOCKET_PUBLIC='true'
WEBSOCKET_PATH="/websocket"
WEBSOCKET_SYSTEM='true'
WEBSOCKET_CORE='true'
WEBSOCKET_CORE_GET='true'
WEBSOCKET_CORE_POST='true'
WEBSOCKET_CORE_PATCH='true'
WEBSOCKET_CORE_DELETE='true'
```

**Comment by umbopepato** [comment]
Looks like the last published version doesn't include the correct env variables mapping (fixed in 0677edb):

<img width="1010" alt="image" src="https://github.com/br41nslug/directus-websocket-subscribe/assets/18363145/e6a248c8-95b7-4035-a615-95f74c15fa61">

**Comment by br41nslug** [comment]
Indeed those granular controls were looking at the wrong env vars 🙈
